src/import/saga.ts

```diff
@@ -159,9 +160,11 @@ export function* importBackupPhraseSaga({ phrase, useEmptyWallet }: ImportBackup
       ValoraAnalytics.track(AppEvents.redux_store_recovery_success, { account })
     }
     ValoraAnalytics.track(OnboardingEvents.wallet_import_success)
+    yield call(initializeAccountSaga)
```

Comment:
i'm awaitng the whole saga here since the `numberVerifiedCentrallySelector` depends on this saga finishing